feat: add Mastodon Support

Topener opened the pull request:
I added my mastodon account. For some reason, a lot of other lines also suddenly changed, which might be due to my formatter, I believe it's a newline issue with mixed newlines (windows vs. mac)

fharper reviewed (changes requested):
Please update your information by modifying `people.json`, not `README.md` which is autogenerated. Instructions are in the README for more details.

Topener commented:
@fharper I completely blanked here, thanks! I changed the PR and updated the script to support Mastodon, and fixed a trailing comma issue for socials.

fharper commented:
Thanks for adding Mastodon, and fixing the rendering issue.
